execute/window_test.go

```diff
@@ -208,6 +208,54 @@ func TestWindow_GetEarliestBounds(t *testing.T) {
 				Stop:  execute.Time(5*time.Minute + 30*time.Second),
 			},
 		},
+		{
+			name: "truncate before offset",
+			w: MustWindow(
+				values.ConvertDurationNsecs(5*time.Second),
+				values.ConvertDurationNsecs(5*time.Second),
+				values.ConvertDurationNsecs(2*time.Second), false),
```

Comment:
Was refreshing my memory on what MustWindow does and noticed that the bool arg months is not used. Should we remove that as part of this change?

Reply:
I did not notice that, yes I'll remove it. I wish the go compiler would identify unused function params.